### PR TITLE
Qr code error level (EXPOSUREAPP-8073)

### PR DESCRIPTION
**Pull request by mtwalli**
- Set default Qr Code error correction level as `Medium`
- Version bump 2.4.2.0

### CODE REVIEW

File: Corona-Warn-App/src/main/res/navigation/nav_graph.xml [hunk 1]
```diff
@@ -815,7 +815,7 @@
             app:argType="string" />
         <argument
             android:name="correctionLevel"
-            android:defaultValue="Q"
+            android:defaultValue="M"
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/navigation/nav_graph.xml:818]: Why did you change this? The change should be about the generated QR Code, not the scanned one.

Comment (harambasicluka) [reply Corona-Warn-App/src/main/res/navigation/nav_graph.xml:818]: @thomasaugsten, @mlenkeit  what do you think?

**Review by harambasicluka — COMMENTED** [review]
not sure about the change on the scanning. But it might help with #3514.

Edit: misunderstood it, it does what it should do!

File: Corona-Warn-App/src/main/res/navigation/nav_graph.xml (hunk 1 above)
Comment (mtwalli) [reply Corona-Warn-App/src/main/res/navigation/nav_graph.xml:818]: @harambasicluka  the qroode should not change when you got to full screen mode

**Review by chiljamgossow — APPROVED** [review]
LGTM

**Review by harambasicluka — APPROVED** [review]
Works 💯

**Review by axelherbstreith — APPROVED** [review]
